partner_gst_qst/views/res_partner_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="res_partner_view_form_inherit_partner_artist" model="ir.ui.view">
+        <field name="name">res.partner.view.form.inherit</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+              <xpath expr="//field[@name='vat']" position="attributes">
+                  <attribute name="invisible">1</attribute>
+             </xpath>
```

Comment:
This module should not make the vat invisible.
We already have a module that does that.
https://github.com/Numigi/odoo-partner-addons/tree/12.0/partner_no_vat